lms/migrations/versions/f521d69a47d2_migration_for_auto_grading_settings.py

```diff
@@ -0,0 +1,60 @@
+"""Migration for auto grading settings."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "f521d69a47d2"
+down_revision = "ff2f8a64f6a9"
+
+
+def upgrade() -> None:
+    op.create_table(
+        "assignment_auto_grading_config",
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column(
+            "activity_calculation",
+            sa.Enum(
+                "cumulative",
+                "separately",
+                name="autogradingcalculation",
+                native_enum=False,
```

Comment:
This is not an enum in postgres, those have been a pain to migrate and manage in the past.